src/psyclone/tests/extract_test.py

```diff
@@ -404,26 +404,16 @@ def test_extract_node_representation(capsys):
     assert schedule.children[1].dag_name == "extract_1"
 
     # Test __str__ method
-    correct = ("""Schedule:
-Loop[]: cell= lower=,,
-kern call: testkern_code
-EndLoop
-ExtractStart
-Loop[]: cell= lower=,,
-kern call: testkern_code
-EndLoop
-Loop[]: cell= lower=,,
-kern call: ru_code
-EndLoop
-ExtractEnd
-Loop[]: cell= lower=,,
-kern call: ru_code
-EndLoop
-Loop[]: cell= lower=,,
-kern call: testkern_code
-EndLoop
-End Schedule""")
-    assert correct in str(schedule)
+    print(str(schedule))
+    assert "\nExtractStart\n" in str(schedule)
```

Comment:
Maybe also include the text that should be immediately before and after the `ExtractStart`/`ExtractEnd` lines.

Reply:
As suggested I added the text before and after, but note that the positioning is also checked with the asserts below.